docs/headings.md

```diff
@@ -20,33 +21,40 @@ hidden: false
 ####Compact Notation
 
 Headers are denoted using a space-separated `#` prefix. While the space is technically required in most standard Markdown implementations, some processors allow for a compact notation as well. ReadMe supports this style, so writing this
-    
-    ###A Valid Heading
-    
-    Lorem ipsum dolor etc.
 
-> 🛑 
+```
+###A Valid Heading
+
+Lorem ipsum dolor etc.
+```
```

Comment:
(Not to be picky since I know this isn't technically within the scope of this PR, but I did notice that this compact header notation isn't working with the new MDX processor!)

Reply:
https://linear.app/readme-io/issue/RM-9896/support-compact-headings